src/app/modules/rup/components/elementos/FormulaBase.html

```diff
@@ -1,45 +1,65 @@
-                    <plex-badge [type]="mensaje.type" *ngIf="mensaje?.texto">
-                        {{ mensaje.texto }}
-                        <plex-button *ngIf="mensaje.add && !soloValores" (click)="addConcepto(mensaje.add)"
-                                     title="Registrar">
-                            <plex-icon name="plus"></plex-icon>
-                        </plex-button>
-                    </plex-badge>
-                </p>
+            <ng-container *ngIf="valorManual">
+                <ng-container *ngFor="let item of elementoRUP.requeridos; let i = index">
+                    <div *ngIf="item.elementoRUP" class="pr-3">
+                        <rup [ngClass]="{'readonly': !valorManual}" [elementoRUP]="item.elementoRUP"
+                             [paciente]="paciente" [params]="item.params" [prestacion]="prestacion"
+                             [registro]="registro.registros[i]" [soloValores]="soloValores" (change)="emitChange2()"
+                             (ejecutarAccion)="emitEjecutarAccion($event)" [habilitado]="valorManual">
+                        </rup>
+                    </div>
+                </ng-container>
             </ng-container>
+
+            <div class="pr-1">
+                <label> {{ registro.concepto.term}} </label>
+
+                <!-- Campo para ingreso manual del valor del registro -->
+                <plex-float *ngIf="!soloValores && valorManual" placeholder="Ingrese un valor"
+                            [(ngModel)]="registro.valor" label='' name="valorCalculo" [readonly]="soloValores"
+                            (change)="emitChange2()" [disabled]="!valorManual">
+                </plex-float>
+                <plex-float *ngIf="!soloValores && !valorManual" [(ngModel)]="registro.valor" required="true"
+                            [disabled]="true">
+                </plex-float>
+                <!-- Valor/resultado del cálculo -->
+                <ng-container *ngIf="soloValores">
+                    <p>
+                        <ng-container *ngIf="!hasRules">
+                            {{registro.valor}}
+                        </ng-container>
+                        <plex-badge [type]="mensaje.type" *ngIf="mensaje?.texto">
+                            {{ mensaje.texto }}
+                            <plex-button *ngIf="mensaje.add && !soloValores" (click)="addConcepto(mensaje.add)"
+                                         title="Registrar">
+                                <plex-icon name="plus"></plex-icon>
+                            </plex-button>
+                        </plex-badge>
+                    </p>
+                </ng-container>
+            </div>
+        </plex-grid>
+        <div class="d-flex justify-content-start align-items-center mt-3">
+            <plex-button *ngIf="!valorManual && !soloValores" type="warning" icon="pencil"
+                         (click)="changeValorManual(true)" tooltip="Agregar manualmente">
```

Comment:
Se corta el tooltip, agregar posición o recortar la frase

```suggestion
            <plex-button *ngIf="!valorManual && !soloValores" type="warning" icon="pencil"
                         (click)="changeValorManual(true)" tooltip="Agregar manualmente" tooltipPosition="left">
```